buckets.yaml

```diff
@@ -1,17 +1,18 @@
-# A mapping of bucket URLs to bucket metadata
+# A mapping of bucket URLs to bucket metadata.
+# Please include a contact for new prefixes!
 #
 # Possible bucket metadata:
 #   prefix: the identifier used for this bucket on the test-history dashboard.
 #   sequential: an optional boolean that indicates whether test runs in this
 #     bucket are numbered sequentially. This is used for an optimization in the
 #     test-history collection phase, and defaults to true.
 
-gs://kubernetes-jenkins/logs/:
+gs://kubernetes-jenkins/logs/:  # @fejta
```

Comment:
is there a reason we don't just make the owner actual metadata? that way we can potentially use it in tooling?